src/rebar_pkg_resource.erl

```diff
@@ -95,7 +95,7 @@ extract(TmpDir, CachePath) ->
     {"metadata.config", Meta} = lists:keyfind("metadata.config", 1, Files),
     {Files, Contents, Version, Meta}.
 
-checksums(Pkg={pkg, _Name, _Vsn, Hash}, Files, Contents, Version, Meta, State) ->
+checksums(Pkg={{pkg, _Name, _Vsn, Hash}, _Registry}, Files, Contents, Version, Meta, State) ->
```

Comment:
Hm? This isn't what it would look like in the lock file if that is what you mean.